### ISSUE TITLE
Not working with Nuxt

### ISSUE THREAD

**Issue by antimodern** [scrumpy/tiptap#152]
I have `Duplicate use of selection JSON ID cell` on first page load and then `render function or template not defined in component: anonymous` errors

```html
<template>
  <div>
    <editor-menu-bar :editor="editor">
      <div slot-scope="{ commands, isActive }">
        <button :class="{ 'is-active': isActive.bold() }" @click="commands.bold">
          Bold
        </button>
       <!--  <button :class="{ 'is-active': isActive.heading({ level: 2 }) }" @click="commands.heading({ level: 2 })">
          H2
        </button> -->
      </div>
    </editor-menu-bar>
    <editor-content :editor="editor" />
  </div>
</template>

<script>
import { Editor, EditorMenuBar, EditorContent } from 'tiptap'
import {
  Bold
} from 'tiptap-extensions'

export default {
  components: {
    EditorMenuBar,
    EditorContent,
  },
  data() {
    return {
      editor: new Editor({
        extensions: [
          new Bold()
        ],
        content: '<p>This is just a boring paragraph</p>',
      }),
    }
  },
  mounted() {
    //
  },
  beforeDestroy() {
    this.editor.destroy()
  }
}
</script>
```

What is wrong? It's pretty confusing that there's no Docs for such complex package.

**Comment by antimodern** [comment]
Lo, now I have third error with same code (when navigate form other route

```
tiptap_extensions__WEBPACK_IMPORTED_MODULE_1__.Bold is not a constructor
```

I guess this is somekind of experimental stuff?

**Comment by antimodern** [comment]
The fix is here - https://github.com/scrumpy/tiptap/issues/131

**Comment by rendisimamora** [comment]
> The fix is here - #131

Work for me. Thanks
